x/bank/keeper/migrations.go

```diff
@@ -1,35 +1,11 @@
 package keeper
 
-import (
-	sdk "github.com/cosmos/cosmos-sdk/types"
-	"github.com/cosmos/cosmos-sdk/x/bank/exported"
-	v2 "github.com/cosmos/cosmos-sdk/x/bank/migrations/v2"
-	v3 "github.com/cosmos/cosmos-sdk/x/bank/migrations/v3"
-	v4 "github.com/cosmos/cosmos-sdk/x/bank/migrations/v4"
-)
-
 // Migrator is a struct for handling in-place store migrations.
 type Migrator struct {
-	keeper         BaseKeeper
-	legacySubspace exported.Subspace
+	keeper BaseKeeper
 }
 
 // NewMigrator returns a new Migrator.
-func NewMigrator(keeper BaseKeeper, legacySubspace exported.Subspace) Migrator {
-	return Migrator{keeper: keeper, legacySubspace: legacySubspace}
-}
-
-// Migrate1to2 migrates from version 1 to 2.
-func (m Migrator) Migrate1to2(ctx sdk.Context) error {
-	return v2.MigrateStore(ctx, m.keeper.storeService, m.keeper.cdc)
```

Comment:
I think this should stay and just return nil (this is what I meant by my other comment).
And we can add a comment saying this migration does not happen from v0.51 or something.

Reply:
what is the reasoning to keep it?

Reply:
So that it is documented in the code why it isn't there and gives a bit of history of how many migrations there has been (but the consensus version basically say that too).
Feel free to skip if you think it shouldn't be there.

Reply:
ill add something there. Cant skip without someones approval 🤷